packages/hydrooj/locales/ko_KR.yaml

```diff
@@ -0,0 +1,821 @@
+Begin at: '시작:'
+Begin Date: 시작 날짜
+Begin Time: 시작 시각
+Belongs to: 소유
+Bio Visibility: 소개 공개
+Bio: 소개
+Blog: Blog
+Bold: 굵게
+Boom!: 펑!
+Browser: 브라우저
+Built-in: 빌트인
+Bulletin: 게시판
+By clicking the button, you will become a member of the domain {0}.: 이 버튼을 클릭하면 귀하는 {0} 의 멤버가 됩니다.
+By Contest: 대회
+By Problem: 문제
+By signing up a Hydro universal account, you can submit code and join discussions in all online judging services provided by us.: Hydro 계정으로 가입하면 Hydro로 제공되는 모든 OJ를 이용하고 참여하실 수 있습니다.
+By Username / UID: 사용자 이름 / UID
+Calendar View: 달력 보기
+Calendar: 달력
+Can be Extended For: '추가 접수:'
+Cancel Score: 채점 취소
+Cancel: 취소
+Categories: 분류
+Category: 분류
+Change Email: 이메일 변경
+Change Password: 비밀번호 변경
+Check In: 로그인
+Chinese: 중국어
+Choose the background image in your profile page.: 프로필 배경을 선택하십시오.
+Claim homework: 숙제 할당
+Claim Homework: 숙제 할당
+Claimed: 할당됨
+Clear Marks: 마크 제거
+Click here to chat with me: 이곳을 눌러 대화
+click to add: 추가
+CLOSE: 닫기
+Code blocks: 코드블럭
+Code copied to clipboard!: 복사 완료
+Code language: 언어
+Code: 코드
+collapse: 병합
+Comment: 댓글
+Comments: 댓글
+CommonMark Syntax: CommonMark Syntax
+Compile Error: 컴파일 에러
+Compilers' Version and Parameters: 컴파일러 버전, 파라미터
+Complete: 완료
+Completed: 완료됨
+Confirm deleting the selected roles?: 정말로 제거하시겠습니까?
+Confirm deleting this comment? Its replies will be deleted as well.: 정말로 제거하시겠습니까? 모든 답글들도 제거됩니다.
+Confirm deleting this reply?: 정말로 제거하시겠습니까?
+Confirm removing the selected users?: 정말로 제거하시겠습니까?
+Confirmation mail has been sent to your new email.: 확인 이메일이 전송되었습니다.
+Congratulations! Your submission is accepted.: 축하합니다! 맞았습니다!
+Contact Us: 연락하세요
+Contact: 연락
+content: 매체
+Content: 매체
+Contest scoreboard is not visible.: 대회 점수판이 비공개입니다.
+Contest Scoreboard: 대회 점수판
+contest_create: 대회 생성
+contest_detail_problem_submit: 대회 상세 문제 제출
+contest_edit: 대회 수정
+contest_main: 대회
+contest_scoreboard: 대회 점수판
+Contest: 대회
+Continue: 계속
+Contributed Problems: 기여한 문제
+Contributions: 기여
+Copy Email: 이메일 복사
+Copy from: 복사
+Copy QQ Number: QQ 계정 복사
+Copy WeChat Account: WeChat 계정 복사
+Copy: 복사
+Copyright Complaint: 저작권 항의
+CPU Info: CPU 정보
+Create a contest: 대회 만들기
+Create a Discussion: 토론 만들기
+Create a problem: 문제 만들기
+Create Contest: 대회 만들기
+Create contests: 대회 만들기
+Create Discussion in {0}: '{0}에 토론 만들기'
+Create Discussion: 토론 만들기
+Create discussions: 토론 만들기
+Create Domain: 그룹 만들기
+Create homework: 숙제 만들기
+Create Homework: 숙제 만들기
+Create Link: 링크 만들기
+Create problem solutions: 해설 만들기
+Create Problem: 문제 만들기
+Create problems: 문제 만들기
+Create Role: 역할 만들기
+Create Training Plan: 학습계획 만들기
+Create training plans: 학습계획 만들기
+Create: 만들기
+Created By: 제작자
+Current dataset comes from: 채점 데이터
+current domain: 그룹
+Current Email: 이메일
+Current Location: 위치
+Current password doesn't match.: 비밀번호가 틀렸습니다.
+Current Password: 현재 비밀번호
+Current Status: 현재 상태
+currently offline: 오프라인
+Data of Problem {0} not found.: 문제 {0}의 데이터가 없습니다.
+Data of problem {1} not found.: 문제 {1}의 데이터가 없습니다.
+Data of record {0} not found.: 평가 {0}의 데이터가 없습니다.
```

Comment:
maybe a space after "{0}"? I'm not sure about Korean, but I thinks a long hexstring ObjectID without a space looks weird.

Reply:
In Korean, it's kind of a suffix. There should be no space. Will it occur problems? (I found some lines without space in zh_CN too.)